client/src/features/projectsV2/fields/RemoveProjectMemberModal.tsx

```diff
@@ -85,7 +86,11 @@ function RemoveProjectMemberAccessForm({
             className={cx("align-items-baseline", "d-flex", "flex-row", "mb-3")}
           >
             <Label>
-              Remove <b>{member.email ?? member.id}</b> from project?
+              Remove <b>{nameToDisplay}</b>{" "}
+              {member.namespace && (
+                <span className="fst-italic">@{member.namespace}</span>
+              )}{" "}
```

Comment:
Building on the change suggested in `roleUtils`: 

```suggestion
              Remove <ProjectMemberDisplay member={member} userInBold={true} />{" "}
```